### ISSUE TITLE
Update to Json.Net 10 planned?

### ISSUE THREAD

**Issue by IceflowRE** [SaladLab/Json.Net.Unity3D#19]

**Comment by veblush** [comment]
Yes, there is a plan. I am waiting for Json.net to be stable because version 10 has many changes.

**Comment by timskap** [comment]
@veblush When update will be finished?

**Comment by jvalenciag** [comment]
When will be the update available?

**Comment by SugoiDev** [comment]
There's a supported JSON.NET package on the asset store. It used to be paid, but the developer made it free after a deal.

The developer from SaladLab (of this integration) has been missing for a long time now, so I don't recommend using it for new projects and, if possible, use the json solution from the asset store.

[PARENTELEMENT, LLC JSON .NET For Unity](https://assetstore.unity.com/packages/tools/input-management/json-net-for-unity-11347)

If you're using the latest Unity, take a look at the forum for possible issues and workarounds: https://forum.unity.com/threads/json-net-for-unity.200336

**Comment by monsdar** [comment]
The version on the Asset Store is based on Newtonsoft.Json 8.0.3 though, so it's even older than this version is. If you're interested here's the source repo of the package: [ianmacgillivray/Json-NET-for-Unity](https://github.com/ianmacgillivray/Json-NET-for-Unity)

**Comment by pfleeter** [comment]
I was trying to upgrade our Unity project to use some libraries that are built for netstandard2.0 and which have Newtonsoft.Json 12.0.1 as a dependency.  Things worked in the editor of course, but when I tried on device with il2cpp builds, I got strange errors like:

`Error getting value from '<our dictionary class member>' on '<our annotated class>'. at Newtonsoft.Json.Serialization.ExpressionValueProvider.GetValue`

I suspected it was maybe something to do with link.xml due to painful past experience, and I found this thread: https://github.com/aspnet/EntityFrameworkCore/issues/13099#issuecomment-420381680

And, at least for our fairly simple use cases, it works!
